### PR TITLE
[CLOUD-4866] Add Jenkinsfile and project infrastructure

### PR DESCRIPTION
**Pull request by satyahegde**
Sets up Jenkins and project infrastructure for forked version of this project

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+podTemplate(
```
Comment (satyahegde) [inline Jenkinsfile:1]: This is copied from https://github.com/ischemaview/cordova-plugin-background-download/blob/development/Jenkinsfile

I don't think any changes are needed here, but I could be wrong.

Comment (alex-rapidai) [reply Jenkinsfile:1]: looks good. thank you.

**Review by alex-rapidai — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
 {
-  "name": "phonegap-plugin-push",
-  "description": "Register and receive push notifications.",
+  "name": "ischemaview-phonegap-plugin-push",
```
Comment (budiirawanrapid) [inline package.json:2]: question: shouldn't the name be `@ischemaview/phone-gap-plugin-push`?

Comment (satyahegde) [reply package.json:2]: I'm not quite sure which way is better. I followed the pattern from `cordova-plugin-background-download` (package.json here: https://github.com/ischemaview/cordova-plugin-background-download), which prepended `ischemaview` to the name of the package but did not include it in the repo. I wanted to stay consistent with the approach in that fork, but I'm open to changing if there is a good reason.

Comment (budiirawanrapid) [reply package.json:2]: got it 👍 maybe for cordova plugin the pattern is different 🤔

**Review by budiirawanrapid — APPROVED** [review]

**Pull request merged**